fitbenchmarking/utils/fitbm_result.py

```diff
@@ -83,6 +83,13 @@ def __init__(
             self.params = controller.final_params[dataset]
             self.accuracy = accuracy[dataset]
 
+        # if SpinW, make sure data_x is correct
+        if self.name == "Triangular AFM Powder Spectrum":
```

Comment:
Similar to in the parser, is there another way we can identify SpinW problems other than using the problem name? Maybe here we could check whether `ebin_cens` is a key in `problem.additional_info`?

Reply:
I've changed this to use ebin_cens as you suggested (this however doesn't work in the case of the parser, so I've just removed the check on the name there)